src/Authentication.ResourceManager/Authentication.ResourceManager.csproj

```diff
@@ -58,14 +58,4 @@
     </EmbeddedResource>
   </ItemGroup>
 
-  <ItemGroup>
-    <Compile Remove="AzureRmServicePrincipalKeyStore.cs" />
-    <Compile Remove="CredStore.cs" />
-    <Compile Remove="Properties\AssemblyInfo.cs" />
-    <Compile Remove="ServicePrincipalKeyStore.cs" />
```

Comment:
I see `AzureRmServicePrincipalKeyStore.cs` removed, but not `ServicePrincipalKeyStore.cs`.

Reply:
There is no ServicePrincipalKeyStore.cs